README.md

```diff
@@ -439,6 +456,10 @@ $ curl -o /path/where/you/want/nancy \
 
 Tests can be run like `go test ./... -v`
 
+Adding new files? Get the license header correct with:
+
+> addlicense -f ./header.txt .
```

Comment:
So this didn't just work for me out of the box..... either we should add to the docs that you have to install `addlicense` (go get -u github.com/google/addlicense) or update the CI build to just do it for you :)

Reply:
I'm a go newb :) I did run that `go get` command, and saw that it updated `go.sum` and `go.mod` and assumed it automatically replicated for other development environments. Wasn't sure how those relate to typical build - will each person that clones the repo need to call that `go get` command for that library specifically?

Reply:
Yes, but we could include it in our `Makefile` more than likely, and just have it run with each build.

Reply:
yeah so in this case since what you want is a binary so you can run it.... `go get` will do 2 things..... 1 install the binary into ~/go/bin and add the libs to your project since we are using `go mod` so that you can use them programmatically. If you run a `go mod tidy` which we do on at CI time.....it'll remove addlicense since its not used by the code. So for us to use it locally ever dev will have to install it manually to run that `addlicense` command above. Thus the CI comment..... more stuff there means no one has to worry about it and it just happens :) But just docs are good too for now. Your call

Reply:
Thanks @zendern - I ran `go mod tidy` and documented the `go get` command, I'll leave the CI changes to @DarthHater and @bhamail ;)

Reply:
@nblair awesome!!! Would you mind writing up an issue to wire this `addlicense` stuff up into CI for us??

Reply:
👍 done in #128 :)